nixos/modules/services/web-apps/dokuwiki.nix

```diff
@@ -266,7 +373,13 @@ in
       "d ${cfg.stateDir}/meta 0750 ${user} ${group} - -"
       "d ${cfg.stateDir}/pages 0750 ${user} ${group} - -"
       "d ${cfg.stateDir}/tmp 0750 ${user} ${group} - -"
-    ];
+      "C ${cfg.usersFile} 0640 ${user} ${group} - ${pkg hostName cfg}/share/dokuwiki/conf/users.auth.php.dist"
+    ] ++ lib.optional (cfg.acl == null) "C ${cfg.aclFile} 0640 ${user} ${group} - ${pkg hostName cfg}/share/dokuwiki/conf/acl.auth.php"
```

Comment:
```suggestion
    ] ++ lib.optional (cfg.acl == null) "C ${cfg.aclFile} 0640 ${user} ${group} - ${pkg hostName cfg}/share/dokuwiki/conf/acl.auth.php.dist"
```

Reply:
See a84c2f4

Reply:
IMO the automatic creation (copying) default user+acl files was nice. Have you found any problems with that?